edgelet/hsm-sys/azure-iot-hsm-c/src/edge_hsm_client_crypto.c

```diff
@@ -68,14 +70,17 @@ void hsm_client_crypto_deinit(void)
     }
     else
     {
-        int status;
-        if ((status = g_hsm_store_if->hsm_client_store_destroy(EDGE_STORE_NAME)) != 0)
+        if (g_crypto_ref == 0)
```

Comment:
Are these changes to support multiple inits of the crypto api?

Reply:
Yes that is right. The multiple init is needed because now the device identity X.509 interface can also use the crypto interface.